build/source/driver/multi_driver.f90

```diff
@@ -247,77 +357,123 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
  case default; call handle_err(30,'unable to identify vegetation category')
 end select
 
+! allocate space for default model parameters
+if(allocated(dparStruct%hru))then
```

Comment:
Why is this check needed? Just for safety reasons (prevent programming errors) or is this something that can actually happen?